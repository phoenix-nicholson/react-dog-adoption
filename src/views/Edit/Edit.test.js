import { render, screen } from '@testing-library/react';
import { MemoryRouter, Route } from 'react-router-dom/cjs/react-router-dom.min';
import DogEdit from './Edit';

test('renders edit form', async () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/dogs/4/edit']}>
      <Route path="/dogs/:id/edit" component={DogEdit} />
    </MemoryRouter>
  );
  await screen.findByDisplayValue('Luna');
  expect(container).toMatchSnapshot();
});
