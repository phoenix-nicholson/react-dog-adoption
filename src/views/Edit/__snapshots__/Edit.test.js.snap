// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders edit form 1`] = `
<div>
  <div>
    <form>
      <label>
        Name:
        <input
          name="name"
          type="text"
          value="Luna "
        />
      </label>
      <label>
        Age:
        <input
          name="age"
          type="text"
          value="4"
        />
      </label>
      <label>
        Image URL:
        <input
          name="image"
          type="text"
          value="https://placedog.net/500?id=101
"
        />
      </label>
      <label>
        Breed:
        <input
          name="breed"
          type="text"
          value="Retriever mix"
        />
      </label>
      <label>
        Bio:
        <input
          name="bio"
          type="text"
          value="Is a Lunatic but sweet"
        />
      </label>
      <button>
        submit
      </button>
    </form>
  </div>
</div>
`;
